etcdserver/v3_server.go

```diff
@@ -558,6 +574,9 @@ func (s *EtcdServer) doSerialize(ctx context.Context, chk func(*auth.AuthInfo) e
 	if err = chk(ai); err != nil {
 		return err
 	}
+
+	trace := traceutil.Get(ctx)
```

Comment:
Should this `trace := traceutil.Get(ctx)` be defined before the line 556 with updated `ctx` with trace key?

```go
trace := traceutil.Get(ctx)
// update ctx
ai, err := s.AuthInfoFromCtx(ctx)
```
?